src/NzbDrone.Core/Localization/Core/en.json

```
@@ -1097,6 +1097,8 @@
   "LogLevel": "Log Level",
   "LogLevelTraceHelpTextWarning": "Trace logging should only be enabled temporarily",
   "LogOnly": "Log Only",
+  "LogSizeLimit": "Log Size Limit",
+  "LogSizeLimitHelpText": "Sets the size in MB above which log files will be automatically archived.",
```

Maybe something like:
```suggestion
  "LogSizeLimitHelpText": "Maximum log file size in MB before archiving. Default is 1MB",
```